scripts/images/README.md

```diff
@@ -13,8 +13,11 @@
 
 ## Prerequisites
 
-- Install python (version 3 and above)
-- Install qemu-img package
+- Install `python3` package
+  - Install `jinja2` & `boto3` modules with `pip3`
+  - Upgrade `PyYAML` module to v5.1 or newer; see https://stackoverflow.com/questions/55551191/module-yaml-has-no-attribute-fullloader
+- Install `qemu-img` package
+- Install 'ibmcloud` CLI; see https://cloud.ibm.com/docs/power-iaas-cli-plugin?topic=power-iaas-cli-plugin-power-iaas-cli-reference
```

Comment:
few instructions are already there in the python file and in the help message as well, having that said - we can have here as well.

Reply:
Glad to hear you agree to have these upfront for smoother operation, instead of getting error and having to hunt for fix.